source/install/desktop-app-changelog.rst

```diff
@@ -20,12 +20,99 @@ Desktop application changelog
 
 Latest Mattermost Desktop App releases:
 
-- `Release v5.0 <#id1>`_
-- `Release v4.7 <#id3>`_
-- `Release v4.6 <#id21>`_
-- `Release v4.5 <#id33>`_
-- `Release v4.4 <#id49>`_
-- `Release v4.3 <#id68>`_
+- `Release v5.1 <#id1>`_
+- `Release v5.0 <#id3>`_
+- `Release v4.7 <#id20>`_
+- `Release v4.6 <#id38>`_
+- `Release v4.5 <#id50>`_
+- `Release v4.4 <#id66>`_
+
+Release v5.1
+--------------
+
+**Release day: 2022-05-16**
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+Compatibility
+~~~~~~~~~~~~~~~
+
+- Desktop Apps are required to be used with any `supported Extended Support Release or a newer Mattermost server version <https://docs.mattermost.com/upgrade/release-lifecycle.html>`_.
+
+Breaking Changes / Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+
+- Added a Desktop App auto-updater. The app now automatically checks for new updates on app start up.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+Linux
+^^^^^^
+
+- Updated the Linux closing behaviour to allow the app to close complely when pressing ``X``.
+- Changed the default setting for **Leave app running in notification area when application window is closed** on Linux to ``false`` by default.
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Added support for Calls for the Desktop App.
```

Comment:
Support for Calls has been in Desktop since the plugin existed as it didn't need special changes on this side. What was added in v5.1 is the ability for the user to select which window to share when screensharing. Previously users could only share the whole screen (including any secondary desktop).